plugins/c8y_configuration_plugin/src/main.rs

```diff
@@ -56,9 +57,6 @@ pub struct ConfigPluginOpt {
 
     #[clap(long = "config-dir", default_value = DEFAULT_TEDGE_CONFIG_PATH)]
     pub config_dir: PathBuf,
-
-    #[clap(long = "config-file", default_value = DEFAULT_PLUGIN_CONFIG_FILE_PATH)]
-    pub config_file: PathBuf,
```

Comment:
I wanted to remove it because in my opinion it is more confusing than helpful. Yes it can be nice to set the path to the configuration file to somewhere else, but then what is `--config-dir` doing anymore? 

For me `--config-dir` is the only place for configuration files. Having a `--config-file` can do more harm than good to the user. Also from our perspective it is more annoying to handle it.

Reply:
In my opinion, `--config-file` option is not good with our `fs_notify` crate. The crate needs to know the directory where the target file exists. 

However, user will give the path to the plugin configuration file by `--config-file`. e.g.
```
c8y_configuration_plugin --config-file '/etc/some/dir/plugin-config.toml
```
Note: with `--config-file` option, even the name of the configuration file has freedom to choose.
 
The config plugin needs to get the directory `/etc/some/dir` and the file name `plugin-config.toml` based on user input. Of course, it is possible to implement, but we need more efforts to support this.

Reply:
Regardless of which direction we will decide or not, the change will be user-faced change. We cannot change it without specification update if we want to change.